src/platform/mod.rs

```diff
@@ -34,12 +40,15 @@ pub trait Platform {
     fn get_max_valid_address() -> usize;
 
     const HAS_S_MODE: bool = config::VCPU_S_MODE;
+    const NB_HARTS: usize = usize::MAX;
 }
 
 pub fn init() {
     Plat::init();
     logger::init();
 
+    log::info!("{:?}", Plat::name());
+
```

Comment:
We can maybe move this log into `main.rs`, we already have a bunch of logs at boot time and this one could go there.

Maybe add some context, like:

```rs
log::info!("Platform: {:?}", Plat::name());
```